diff --git a/pint/quantity.py b/pint/quantity.py
index 747432c..09f7aed 100644
--- a/pint/quantity.py
+++ b/pint/quantity.py
@@ -1548,7 +1548,7 @@ class Quantity(PrettyIPython, SharedRegistryObject):
         else:
             return value
 
-    def __array__(self):
+    def __array__(self, t=None):
         warnings.warn(
             "The unit of the quantity is stripped when downcasting to ndarray.",
             UnitStrippedWarning,
